### ISSUE TITLE
Patches from 3rd party repos not loaded

### ISSUE THREAD

**Issue by danielvijge** [LMS-Community/slimserver#51]
Patches from 3rd party repositories are not loaded in 7.9. In 7.8.1 these patches are still loaded.

Steps to reproduce:
- Have a Squeezebox Touch with Patch Installer available
- On the LMS settings, go to Plugins and add a test repo http://server.vijge.net/static/squeezebox/test.xml
- On the Squeezebox Touch, open the Patch Installer settings

When connected to a 7.8.1 server, "Test patch" appears in the list. When connected to a 7.9.0, "Test patch" does not appear in the list.

I tested this with 7.8.1 & 7.9.0 nightlies from May 23, running on OSX. But the same happens with my main server, now running 7.9.0@April-10 on Debian armhf.

**Comment by mherger** [comment]
What if you set maxTarget to 7.9.\* or \* right away?

**Comment by danielvijge** [comment]
Nope, still doesn't show up. I also have a patch that is included in the official extension list (http://repos.squeezecommunity.org/extensions.xml - ReduceBrightness), and here the maxTarget is set to 7.9. This patch does show up. So this only applies to patches in 3rd party repos.

**Comment by mherger** [comment]
What device are you testing with? ReduceBrightness is only targeted at fab4 (Touch), but eg. not the Radio.

**Comment by mherger** [comment]
Reduce Brightness does show up on my Touch. It wouldn't on Radio, as it's limited to fab4 according to the repository file.

All other patches fail, because the Touch is running a 7.8.x firmware, while the patches have a lower maxTarget. If you enable debugging for plugin.extensions (in LMS) you'll see all of this information.

**Comment by danielvijge** [comment]
I tested it on a Touch. The test patch referenced above is also for the Touch, but does not show up.
I started investigating because a user on the forums mentioned that he could not install ReduceBrightness on a Radio, even after adding the beta repo, which does contain the patch for the Radio.
On 26 May 2015 10:36, mherger notifications@github.com wrote:Reduce Brightness does show up on my Touch. It wouldn't on Radio, as it's limited to fab4 according to the repository file.

All other patches fail, because the Touch is running a 7.8.x firmware, while the patches have a lower maxTarget. If you enable debugging for plugin.extensions (in LMS) you'll see all of this information.
